src/com/epam/collections/models/Student.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.collections.models;
+
+public class Student implements Comparable {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private int age;
+
+    public Student(int id, String firstName, String lastName, int age) {
+        this.id = id;
```

Comment:
Do we really need `id`?

Reply:
Suppose  there are two students whose firstnames ,lastnames and ages are the same .Thats why i added ID ,to differ them.

Reply:
You are correct, we need a way to make Students unique. For current exercise, we suppose first name, last name and age are enough. Introducing ID will definitely make Students more unique but then you need to create much more complicated mechanisms for storing and retrieving those IDs globally, e.g. whenever and wherever in the project you create a student, you should know which ID is not used yet.